supervisor/pip.sls

```diff
@@ -2,7 +2,7 @@ supervisor:
   pip.installed:
     - name: supervisor==3.0
     - require:
-      - pkg: python-pip
+      - pip: pip
```

Comment:
Should we `include` the python statefile at the top of this file? Seems we might not need it since it was working without it, but not sure if that is just an accident :)